### ISSUE TITLE
utf8 - ndash in grammar not recognised at runtime

### ISSUE THREAD

**Issue by ptica** [Nordth/php-pegjs#2]
if I include a n-long dash in the grammar  ("–") I'm getting a parse error that "\u2013" is expected but "–" was found

```
Syntax error: Expected " ", "\u2013" or end of input but "–" found. At line 1 column 7 offset 6
```

I'm not that familiar with php support for unicode, what should I do to parse input with "–"?

**Comment by ptica** [comment]
If I replace the occurrences  of "\u2013" with "–" in the source of generated parser it works ok then

**Comment by ptica** [comment]
the unicode code point is generated by utils.quote here:
https://github.com/Nordth/php-pegjs/blob/master/src/passes/generate-bytecode-php.js#L560
which uses the original js version of utils.escape from
https://github.com/pegjs/pegjs/blob/v0.8.0/lib/utils.js#L185

not really sure whether `php` could be convinced into using the unicode code points -- or the grammar should be generated rather without them -- I'll leave the issue here for someone more knowledgable

**Comment by Nordth** [comment]
Possible fixed by 186f0bd
